pages/datasets/_datasetId.vue

```diff
@@ -322,23 +327,23 @@ export default {
      * @returns {String}
      */
     datasetLicenseName: function() {
-      return propOr('', 'license', this.datasetDetails)
+      return propOr('', 'license', this.datasetInfo)
```

Comment:
I see this pattern in a lot of places and IMHO it isn't always necessary.  

This approach is "safe" in that you will never get an undefined error.  The tradeoff is that it is confusing.  Can `datasetInfo.banner` be undefined?  Can `datasetInfo` itself be undefined?  If only `banner` can be undefined, this is much clearer: `this.datasetInfo.banner || ''`.  If banner is never undefined, then just saying `this.datasetInfo.banner` is much easier to read when one is unfamiliar with the code.

No need to change, just my two cents on style again :)

Reply:
This is a great point @matthewtgilbride thank you for sharing :) I've usually adopted this pattern because it was consistently used in the codebase when I started working on the team, but I think it makes sense to apply this for readability purposes. I know I was confused with reading the code when I first started and had no idea what ramda was haha.

Reply:
```
    /**
     * Returns the dataset storage count
     * @returns {Object}
     */
    datasetStorage: function() {

      const [number, unit] = this
        .formatMetric((this.datasetInfo || {}).size || 0)
        .split(' ')

      return {
        number,
        unit
      }

    },
```